File: sort.c
#include <stdlib.h>
#include <stdio.h>

#include "time.h"
int main(int arg,char **argv)
{
    clock_t start_time = clock();
    int data[3000];
    int i;
    int length=sizeof(data)/sizeof(int);
    for(i=0; i<length;i++)
    {
        data[i] = rand()%10000+1;
    }
    int j,temp;
    for(i=0;i<length;i++)
    {
        for(j=0;j<length-1-i;j++)
        {
            if(data[j] > data[j+1])
            {
                temp = data[j];
                data[j] = data[j+1];
                data[j+1] = temp;
            }
        }
    }

   /* for(i=0;i<length;i++)
    {
        printf("%d\n",data[i]);
    }*/
    clock_t end_time = clock();
    float  time = (double)(end_time-start_time)/CLOCKS_PER_SEC;
    printf("use time: %f",time);
    return 0;
}

File: sort.php
<?php
    $start_time =microtime(true);
    $data =array();
    for($i=0; $i<3000; $i++)
    {
        array_push($data,rand(0,10000));
    }
//    var_dump($data);
    $temp;
    for($i=0;$i<count($data);$i++)
    {
        for($j=0;$j<count($data)-1-$i;$j++)
        {
            if($data[$j] > $data[$j+1])
            {
                $temp = $data[$j];
                $data[$j] = $data[$j+1];
                $data[$j+1] = $temp;
            }
        }
    }
//    var_dump($data);
    $end_time =microtime(true);
    $time = $end_time-$start_time;
    echo "use time:" ,$time;
